### PR TITLE
Release notes for PS 5.5.57-38.9

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+in the `5.5.57-38.9 milestone on Launchpad
+<https://launchpad.net/percona-server/+milestone/5.5.57-38.9>`_.
+
+.. note:: Red Hat Enterprise Linux 5 (including CentOS 5 and other derivatives),
+   Ubuntu 12.04 and older versions are no longer supported by Percona software.
+
+New Features
+============
+
+* Added support and packages for Debian 9 (stretch).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.57-38.9.rst]: Please consult https://github.com/percona/percona-server/pull/1851/files and https://github.com/percona/percona-server/pull/1854/files for feature and bug bug descriptions

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  in SSL certificates for ``--ssl-verify-server-cert``.
+  For more information, see :ref:`compatibility-matrix`.
+
+* :bug:`1705729`: Fixed the ``postinst`` script
+  to correctly locate the ``datadir``.
+
+* :bug:`1709834`: Fixed the ``mysqld_safe`` script
+  to correctly locate the ``basedir``.
+
+* Minor fixes: :bug:`1160986`, :bug:`1622985`, :bug:`1684601`, :bug:`1689998`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.57-38.9.rst]: 1622985 should be documented

File: doc/source/upstream-bug-fixes.rst [hunk 3]
```diff
@@ -145,31 +163,31 @@
...
 |:Launchpad bug: :bug:`1401791`                                                                               |
-|:Upstream state: Verified (checked on 2017-05-09)                                                            |
+|:Upstream state: Verified (checked on 2017-08-11)                                                            |
 |:Fix Released: :rn:`5.5.41-37.0`                                                                             |
 |:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`74440` - mysql_install_db not handling mysqld startup failure                      |
 |:Launchpad bug: :bug:`1382782`                                                                               |
-|:Upstream state: Won't Fix                                                                                   |
+|:Upstream state: Won't fix                                                                                   |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst:184]: Spurious change?

File: doc/source/upstream-bug-fixes.rst [hunk 4]
```diff
@@ -5,9 +5,27 @@
 =============================================================
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst:5]: Upstream bug 83264 state should have changed with this releae

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst (hunk 1 above)
Comment (azhebel) [reply doc/source/release-notes/Percona-Server-5.5.57-38.9.rst]: @laurynas-biveinis the first link has no features. Everything from the second one is reflected. What didn't I cover relevant to 5.5? It should be mentioned here https://launchpad.net/percona-server/+milestone/5.5.57-38.9

Comment (laurynas-biveinis) [reply doc/source/release-notes/Percona-Server-5.5.57-38.9.rst]: @azhebel, re. features not much, just that Debian 9 packages are only available for amd64

File: doc/source/upstream-bug-fixes.rst (hunk 4 above)
Comment (azhebel) [reply doc/source/upstream-bug-fixes.rst:5]: It is not listed here https://www.percona.com/doc/percona-server/5.5/upstream-bug-fixes.html
What is the corresponding LP bug?

File: doc/source/upstream-bug-fixes.rst (hunk 3 above)
Comment (azhebel) [reply doc/source/upstream-bug-fixes.rst:184]: It was a conflict with earlier change

File: doc/source/upstream-bug-fixes.rst (hunk 4 above)
Comment (laurynas-biveinis) [reply doc/source/upstream-bug-fixes.rst:5]: https://bugs.launchpad.net/percona-server/+bug/1628417

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst (hunk 2 above)
Comment (azhebel) [reply doc/source/release-notes/Percona-Server-5.5.57-38.9.rst]: 👍

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst (hunk 1 above)
Comment (azhebel) [reply doc/source/release-notes/Percona-Server-5.5.57-38.9.rst]: Added

File: doc/source/upstream-bug-fixes.rst (hunk 4 above)
Comment (azhebel) [reply doc/source/upstream-bug-fixes.rst:5]: Added

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**